ftw/showroom/js/src/register.js

```diff
@@ -27,12 +27,16 @@ export default function Register(items = [], options) {
   function checkPointer() {
     oberserver.update(pointer);
     if(oberserver.hasChanged()) {
-      if (pointer === 0) {
-        options.head(reveal.current);
-      }
-      if (pointer === reveal.size - 1) {
-        options.tail(reveal.current);
-      }
+      performCalls();
+    }
+  }
+
+  function performCalls() {
+    if (pointer === 0) {
+      options.head(reveal.current);
+    }
+    if (pointer === reveal.size - 1) {
+      options.tail(reveal.current);
```

Comment:
same here, this is a property which returns the current item (https://github.com/4teamwork/ftw.showroom/blob/master/ftw/showroom/js/src/register.js#L72)